src/containerapp/azext_containerapp/tests/latest/test_containerapp_session_code_interpreter_nodelts.py

```diff
@@ -119,10 +109,5 @@ def test_containerapp_session_code_interpreter_nodelts_e2e(self, resource_group)
             resource_group,
             ))
         
-        self.cmd("containerapp sessionpool delete -n {} -g {} --yes".format(
```

Comment:
why remove the deletion code?

Reply:
This is to remove the deletion of that mistakenly created PythonLTS session pool https://github.com/Azure/azure-cli-extensions/pull/8249/files/dd6dc3deae0818beb7408a423208e136fd4d31a2#r1836222120